### ISSUE TITLE
Add Bump in “Next Window” settings

### ISSUE THREAD

**Issue by Bindou01** [jurialmunkey/skin.arctic.fuse#678]
### Skin section

Widgets

### Feature description

Hi Jurial, can you add the Bump function in the “Next Windows” Widgets settings?

### Why is this feature needed? Please justify and explain.

This would allow you to put the "Filter/wall" function in the media library, and leave a bump in the Widgets. :D

### Checklist

- [X] I have made a separate issue for each feature I am requesting.
- [X] I have searched the issues and this feature has not been requested.
- [X] I am using the most recent skin version.

**Comment by jurialmunkey** [comment]
Filter in library will bump on widgets

**Comment by Bindou01** [comment]
> Filter in library will bump on widgets

Indeed, I just tried and it works as I hoped, there is a Bump in the Widgets and the filter present in the Media Library. Excellent ! Thank you so much !

And I just saw that you managed to fix all the issues with the Search hub. Everything works absolutely perfectly. This hub is killer now. I am also surprised because you indicated that it is not possible to correct the problem with the return key. Perhaps we misunderstood each other. In any case, everyone here is delighted! Fuse is now 100% functional on our side.
